lib/components/TokenDropdown.jsx

```diff
@@ -10,7 +12,7 @@ import UsdcSvg from 'assets/images/usdc.svg'
 import UsdtSvg from 'assets/images/usdt.svg'
 import WbtcSvg from 'assets/images/wbtc.svg'
 import ZrxSvg from 'assets/images/zrx.svg'
-import { WalletContext } from 'lib/components/WalletContextProvider'
+import RepSvg from 'assets/images/rep.svg'
```

Comment:
Is Rep something we need to use on Kovan? Compound's site says REP (Augur) is deprecated
<img width="1018" alt="Screenshot 2021-03-09 at 14 55 29" src="https://user-images.githubusercontent.com/35717/110549828-7f82a080-80e7-11eb-9cb3-5fc45e53d6e9.png">